apps/omg/config/config.exs

```diff
@@ -8,8 +8,14 @@ config :omg,
   coordinator_eth_height_check_interval_ms: 6_000,
   metrics_collection_interval: 60_000,
   input_pointer_types_modules: %{<<1>> => OMG.InputPointer.UtxoPosition},
-  output_types_modules: %{<<1>> => OMG.Output.FungibleMoreVPToken},
-  tx_types_modules: %{<<1>> => OMG.State.Transaction.Payment}
+  output_types_modules: %{
+    <<1>> => OMG.Output.FungibleMoreVPToken,
+    <<2>> => OMG.Output.FungibleMoreVPToken
+  },
+  tx_types_modules: %{
+    <<1>> => OMG.State.Transaction.Payment,
+    <<3>> => OMG.State.Transaction.FeeTokenClaim
+  }
```

Comment:
it feels very awkward to have to determine the module needed via application level configuration like this... While I see there's already history of this happening, I think this needs to be pushed down into the code to make that determination 🤔

Reply:
Agreed, can't we just move this to the actual modules? Same way it's being done in `ex_plasma` I think.

Reply:
Yeah - we could figure out a nicer way - any suggestion is welcomed & appreciated ❗️